libs/src/services/hedgehog/Hedgehog.ts

```diff
@@ -25,7 +31,8 @@ export class Hedgehog {
   constructor({
     identityService,
     useLocalStorage = true,
-    localStorage
+    localStorage,
+    createKey = getPlatformCreateKey()
```

Comment:
Oh so we need to default here too bc it's passed to `createAuthLookupKey`?

Reply:
yeah exactly, since we are overriding functionality we need some way to get it in case they dont provide it